Specs/Renderer/BufferSpec.js

```diff
@@ -11,685 +11,690 @@ defineSuite([
         createContext) {
     'use strict';
 
-    var context;
-    var buffer;
-    var buffer2;
-
-    beforeAll(function() {
-        context = createContext();
-    });
-
-    afterAll(function() {
-        context.destroyForSpecs();
-    });
-
-    afterEach(function() {
-        if (buffer && !buffer.isDestroyed()) {
-            buffer = buffer.destroy();
-        }
-        if (buffer2 && !buffer2.isDestroyed()) {
-            buffer2 = buffer2.destroy();
-        }
-    });
-
-    it('throws when creating a vertex buffer with no context', function() {
-        expect(function() {
+    createBufferSpecs();
+    createBufferSpecs({ defaultToWebgl2 : true });
```

Comment:
Can you check if WebGL 2 is not supported and then skip these tests so they are not ran twice with WebGL 1?